src/transformers/generation/logits_process.py

```diff
@@ -1517,29 +1518,33 @@ class WhisperTimeStampLogitsProcessor(LogitsProcessor):
     ```
     """
 
-    def __init__(self, generate_config):  # support for the kwargs
+    def __init__(
```

Comment:
This logit processor was incorrectly added before. Also note that since we never had support for "sequential" or "timestamp"-chunked transcription before, we didn't really use the class before. It was only used when single 30s chunks were supposed to predict timestamps.